frontend/packages/kubevirt-plugin/src/components/vm-templates/instantiate-template/TemplateForm.tsx

```diff
@@ -157,7 +187,10 @@ export const TemplateForm: React.FC<TemplateFormProps> = ({
             }: TemplateParameter) => {
               const paramValue = parameters[name] || '';
               const helpID = description ? `${name}-help` : '';
-              const placeholder = generate ? '(generated if empty)' : '';
+              const placeholder =
+                generate || (missingParams.find(([key]) => key === name) && !requiredParam)
```

Comment:
help text should be "(set to <value> if empay)"

Reply:
(set to if [value] empay)"

Reply:
`(set to [value] if left empay)`

Reply:
fixed